### DISCUSSION TITLE
nutgram:listen command

### DISCUSSION THREAD

**Discussion by abbasudo** [nutgram/nutgram#711] [Ideas]
Hello.
Thanks for your awesome package. it saved me a lot of time. 
`nutgram:run` command In laravel doesn't get updated upon change on the app. Currently, I have to restart the command each time I make changes in the app when developing. It would be nice if we had a `nutgram:listen` or `nutgram:watch` command to run the app while watching changes. that comes in handy when developing.

**Comment by abbasudo** [comment]
I have created this feature in my local project. I can make a PR If you agree with this feature.

**Reply by sergix44** [reply]
If you want us to take a look feel free to open a PR 😄 
Can't guarantee the merge before taking a look at the code
